### ISSUE TITLE
Name field missing in Customer details.

### ISSUE THREAD

**Issue by malwarebo** [razorpay/razorpay-php#155]
Looks like among the values passed in **prefill**, only _contact_ and _email_ are stored. **name** is missing when I check **Customer** details in dashboard or access using the API. What is the issue here or is it that **name** is not sent at all?

**Comment by harman28** [comment]
There's some confusion here. The prefill values you're passing to checkout only control what your payer sees on checkout, and allows them to skip entering these details themselves. They do not actually create `customer` entities on your dashboard. Those can only be created by you, using [these APIs](https://razorpay.com/docs/api/customers/).

The name you're passing is actually getting used as cardholder name, so it won't get used at all if the payment is not done via card.

In summary: use prefill for easing checkout experience for the payer. Use customers API if you have [other use cases](https://razorpay.com/docs/customers/).
